test/monitoring/monitoring.go

```diff
@@ -74,15 +77,31 @@ func PortForward(logf logging.FormatLogger, podList *v1.PodList, localPort, remo
 }
 
 // RunBackground starts a background process and returns the Process if succeed
-func executeCmdBackground(logf logging.FormatLogger, stdout io.Writer, stderr io.Writer, format string, args ...interface{}) (*os.Process, error) {
+func executeCmdBackground(logf logging.FormatLogger, format string, args ...interface{}) (*os.Process, error) {
 	cmd := fmt.Sprintf(format, args...)
 	logf("Executing command: %s", cmd)
 	parts := strings.Split(cmd, " ")
 	c := exec.Command(parts[0], parts[1:]...) // #nosec
-	c.Stdout = stdout
-	c.Stderr = stderr
+	outPipe, err := c.StdoutPipe()
+	if err != nil {
+		return nil, fmt.Errorf("%s cannot get stdout pipe: %w", cmd, err)
+	}
+	go printPipe(logf, outPipe)
+	errPipe, err := c.StderrPipe()
+	if err != nil {
+		return nil, fmt.Errorf("%s cannot get stderr pipe: %w", cmd, err)
+	}
+	go printPipe(logf, errPipe)
 	if err := c.Start(); err != nil {
 		return nil, fmt.Errorf("%s command failed: %w", cmd, err)
 	}
 	return c.Process, nil
 }
+
+func printPipe(logf logging.FormatLogger, reader io.ReadCloser) {
+	defer reader.Close()
+	scanner := bufio.NewScanner(reader)
+	for scanner.Scan() {
```

Comment:
if you want to be fancy
```suggestion
	
	for scanner := bufio.NewScanner(reader); scanner.Scan(); {
```

Reply:
I prefer two lines :)